### PR TITLE
Docs - Add result contributing rules

### PR DESCRIPTION
**Pull request by TobeyQin**
**Description**
Add result contributing rules

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: docs/developer-guides/contributing.md [hunk 1]
```diff
@@ -38,3 +40,52 @@ If you would like to contribute to the project, please follow below steps of joi
...
+            └── systems
+                ├── system1.json
+                └── system2.json
+
+#### Files to provide
+
+Besides `README` and `LICENSE` file, you should provide at least three benchmarking related files.
+* `system.json`: This file lists all the system configuration list in json format.
+* `config.yaml`: This file is the config file to run benchmarks.
+* `result.json`: This file contains the results run by SuperBench with system configuation listed in `system.json` file.
```
Comment (cp5555) [inline docs/developer-guides/contributing.md:87]: Do we need to give details for these files?

Comment (TobeyQin) [reply docs/developer-guides/contributing.md:87]: I think we can add the detail example later when we generate the initial results.

**Review by cp5555 — APPROVED** [review]

File: docs/developer-guides/contributing.md [hunk 2]
```diff
@@ -38,3 +40,50 @@ If you would like to contribute to the project, please follow below steps of joi
...
+
+If you want to contribute benchmark results run by specified SuperBench version, please follow below guidelines.
+
+#### Where to submit
+
+All the results are stored under `results` folder. The directory structure is as follows. Please create `<your-benchmark-folder>` to submit results.
+
+```
+.
+└── results
```
Comment (abuccts) [inline docs/developer-guides/contributing.md:55]: remove `.`?

File: docs/developer-guides/contributing.md [hunk 3]
```diff
@@ -38,3 +40,50 @@ If you would like to contribute to the project, please follow below steps of joi
...
+            └── systems
+                ├── system1.json
+                └── system2.json
+```
+
+#### Files to provide
+
+Besides `README` and `LICENSE` file, you should provide at least three benchmarking related files.
+* `system.json`: This file lists all the system configurations in json format.
+* `config.yaml`: This file is the config file to run benchmarks. Click [here](https://microsoft.github.io/superbenchmark/docs/getting-started/configuration) to learn the details.
```
Comment (abuccts) [inline docs/developer-guides/contributing.md:88]: use relative path for md link
```suggestion
* `config.yaml`: This file is the config file to run benchmarks. Click [here](../getting-started/configuration) to learn the details.
```

**Review by abuccts — APPROVED** [review]

File: docs/developer-guides/contributing.md (hunk 2 above)
Comment (TobeyQin) [reply docs/developer-guides/contributing.md:55]: sure.

File: docs/developer-guides/contributing.md (hunk 3 above)
Comment (TobeyQin) [reply docs/developer-guides/contributing.md:88]: fixed

**Pull request merged**